Jenkinsfile

```diff
@@ -287,6 +287,14 @@ try {
               Utils.markStageSkippedForConditional("tpchQueryPlansAndVerification")
             }
           }
+        }, tpchQ15QueryPlans: {
+          stage("tpchQ15QueryPlans") {
+            if (env.BRANCH_NAME == 'master' || full_ci) {
+              sh "mkdir -p query_plans/tpch; cd query_plans/tpch; ln -s ../../resources; ../../clang-release/hyriseBenchmarkTPCH -r 1 -q 15 --visualize"
```

Comment:
This might cause concurrency issues with a second binary running in the same folder. Why not just add the `sh` command to the existing stage?

Reply:
Ok, good idea ;)